android/src/main/java/com/artirigo/kontaktio/BeaconProximityManager.java

```diff
@@ -38,6 +41,13 @@ ProximityManager init(ReadableArray beaconTypes) throws Exception {
         // create manager
         proximityManager = ProximityManagerFactory.create(reactAppContext);
 
+        // this forces the new Anchor 2 beacons to include `uniqueId` with beacon ranging data
+        String proximityUuid = String.valueOf(KontaktSDK.DEFAULT_KONTAKT_BEACON_PROXIMITY_UUID);
```

Comment:
Optimally, this should be possible to configure through the library's API as some consumers of the lib might not use the default kontakt beacon proximity UUID and also some might not want to resolve UUID during ranging for some reason (I believe Kontakt didn't add it by default for some security reason but I'm not certain).